src/build_system.ml

```diff
@@ -212,7 +212,21 @@ module File_spec = struct
 end
 
 module Alias0 = struct
-  type t = { dir : Path.t; name : string }
+  include
```

Comment:
I find this syntax slightly heavy. What about `module M : sig ... end = struct ... end include M`?

Reply:
Okay. I'll use `T` over `M` since that's what we do elsewhere as well.